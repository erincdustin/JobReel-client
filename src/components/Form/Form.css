

.Input {
  border-radius: 10px;
  width: 15em;
  height: 1.5em;
}