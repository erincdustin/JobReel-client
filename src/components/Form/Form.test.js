import React from 'react'
import ReactDOM from 'react-dom'
import { Input, Label, Required, Textarea } from './Form'

describe('Form smoke tests', () => {
  describe('Input smoke test', () => {
    it('renders without crashing', () => {
      const div = document.createElement('div')
      ReactDOM.render(<Input />, div)
      ReactDOM.unmountComponentAtNode(div)
    })
  })

  describe('Label smoke test', () => {
    it('renders without crashing', () => {
      const div = document.createElement('div')
      ReactDOM.render(<Label />, div)
      ReactDOM.unmountComponentAtNode(div)
    })
  })

  describe('Required smoke test', () => {
    it('renders without crashing', () => {
      const div = document.createElement('div')
      ReactDOM.render(<Required />, div)
      ReactDOM.unmountComponentAtNode(div)
    })
  })

  describe('Textarea smoke test', () => {
    it('renders without crashing', () => {
      const div = document.createElement('div')
      ReactDOM.render(<Textarea />, div)
      ReactDOM.unmountComponentAtNode(div)
    })
  })
})
