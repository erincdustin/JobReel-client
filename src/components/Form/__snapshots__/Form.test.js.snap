// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form smoke tests Input snapshot test renders the UI as expected 1`] = `
<input
  className="Input"
  type="text"
/>
`;

exports[`Form smoke tests Label snapshot test renders the UI as expected 1`] = `
<label
  className="Label"
/>
`;

exports[`Form smoke tests Required snapshot test renders the UI as expected 1`] = `
<span
  className="Required"
>
  *
</span>
`;

exports[`Form smoke tests Textarea snapshot test renders the UI as expected 1`] = `
<textarea
  className="Textarea"
/>
`;
